src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/bc_on_app_capability_updated_notification_test.cc

```diff
@@ -0,0 +1,125 @@
+namespace {
+const uint32_t kConnectionKey = 1u;
+}  // namespace
+
+MATCHER_P(CheckAppCapability, app_capability, "") {
+  return app_capability == (*arg)[strings::msg_params][strings::app_capability];
+}
+
+class BCOnAppCapabilityUpdatedNotificationTest
+    : public CommandsTest<CommandsTestMocks::kIsNice> {
+ protected:
+  void SetUp() OVERRIDE {
+    message_ = CreateMessage();
+    (*message_)[strings::params][strings::connection_key] = kConnectionKey;
+    command_ = CreateCommand<BCOnAppCapabilityUpdatedNotification>(message_);
+    mock_app_ = CreateMockApp();
+  }
+
+  void FillVideoStreamingCapability(
+      smart_objects::SmartObject& video_streaming_capability) {
+    video_streaming_capability[strings::preferred_resolution] =
+        smart_objects::SmartObject(smart_objects::SmartType_Map);
+    video_streaming_capability[strings::preferred_resolution]
+                              [strings::resolution_width] = 800;
+    video_streaming_capability[strings::preferred_resolution]
+                              [strings::resolution_height] = 354;
+    video_streaming_capability[strings::max_bitrate] = 10000;
+    video_streaming_capability[strings::supported_formats] =
+        smart_objects::SmartObject(smart_objects::SmartType_Array);
+    video_streaming_capability[strings::supported_formats][0] =
+        smart_objects::SmartObject(smart_objects::SmartType_Map);
+    video_streaming_capability[strings::supported_formats][0]
+                              [strings::protocol] =
+                                  hmi_apis::Common_VideoStreamingProtocol::RAW;
+    video_streaming_capability[strings::supported_formats][0][strings::codec] =
+        hmi_apis::Common_VideoStreamingCodec::H264;
+    video_streaming_capability[strings::haptic_spatial_data_supported] = true;
+    video_streaming_capability[strings::diagonal_screen_size] = 7.47;
+    video_streaming_capability[strings::pixel_per_inch] = 117.f;
+    video_streaming_capability[strings::scale] = 1.f;
+  }
+
+  BCOnAppCapabilityUpdatedNotificationPtr command_;
+  MockAppPtr mock_app_;
+  MessageSharedPtr message_;
+};
+
+TEST_F(BCOnAppCapabilityUpdatedNotificationTest, Run_SendMessageToHMI_SUCCESS) {
+  smart_objects::SmartObject app_capability =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  app_capability[strings::app_capability_type] =
+      mobile_apis::AppCapabilityType::VIDEO_STREAMING;
+  app_capability[strings::video_streaming_capability] =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  FillVideoStreamingCapability(
+      app_capability[strings::video_streaming_capability]);
+
+  (*message_)[strings::msg_params][strings::app_capability] = app_capability;
+
+  ASSERT_TRUE(command_->Init());
+
+  EXPECT_CALL(mock_rpc_service_,
+              SendMessageToHMI(CheckAppCapability(app_capability)));
+  command_->Run();
+}
+
+}  // namespace bc_on_app_capability_updated_notification_test
+}  // namespace mobile_commands_test
```

Comment:
```suggestion
}  // namespace hmi_commands_test
```

Reply:
Done in 8212d6309933c271ae7908e3b457869a9b0d5468